### PR TITLE
D2IQ-64140: on prem alpha tutorial

### PR DESCRIPTION
**Pull request by nLight**
https://jira.d2iq.com/browse/D2IQ-64140

## Description of changes being made
1. Tutorial how to run a cluster on premise based of https://docs.google.com/document/d/1Wd-ckUtZWAAVXWH2fChK1UuphbDaDiaXQrjNEABroMQ/edit
2. Called out on premise provider is Alpha
3. adjusted some heading levels
4. updated some related Cloud Providers screenshots

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/latest/tutorials/on-prem-alpha/index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+layout: layout.pug
+navigationTitle: Launching a cluster On Premise
+title: Launching a cluster On Premise (Alpha)
+preview: true
+excerpt: Use a previously deployed Konvoy cluster to deploy a new Konvoy cluster with Kommander On Premise Provider
+---
+
+## Previous Setup
```
Comment (samvantran) [inline pages/ksphere/kommander/latest/tutorials/on-prem-alpha/index.md]: Previous step? Should this be `Pre-requisites`?

Comment (russurquhart1) [reply pages/ksphere/kommander/latest/tutorials/on-prem-alpha/index.md]: I agree, I think that should be Prerequisite.

Comment (nLight) [reply pages/ksphere/kommander/latest/tutorials/on-prem-alpha/index.md]: I copied that from `Duplicating a Production Cluster` tutorial 🙈

**Comment by nLight** [comment]
I addressed all comments and Russ fixed my broken english 🙇 
I think we can lift the WIP label.

**Pull request merged**